style.css

```diff
@@ -137,3 +138,351 @@ body {
   height: 20px;
   width: 18px;
 }
+
+.card {
+  padding: 16px 16px 25px 16px;
+  background: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.card__content--heading {
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+}
+
+.card__item--heading {
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  text-transform: uppercase;
+  font-weight: 800;
+  list-style: none;
+}
+
+.card__content--item1 {
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+  gap: 15px;
+  color: #7a869a;
+}
+
+.card__content--item2 {
+  display: flex;
+  align-items: center;
+  list-style-type: none;
+  gap: 15px;
+  margin-bottom: 10px;
+}
+
+.card__content--desc {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.tech__btn {
+  padding: 4px 12px;
+  background: #ebebff;
+  border-radius: 8px;
+  color: #396df2;
+}
+
+.btn {
+  padding: 16px;
+  background: transparent;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  color: #396df2;
+}
+
+.works {
+  display: flex;
```

Comment:
- [ ] Please take note that you will have to display this `main` container as `grid` according to the requirement. 👍